v1.0/docs/admin/kube-apiserver.md

```diff
@@ -1,5 +1,6 @@
 ---
 layout: docwithnav
+title: admin/kube-apiserver.md
```

Comment:
Empirically we have several docs that have no single # but do have a double # which appears to be a title.  We could fix those docs, I suppose...